### PR TITLE
New package: cookie-0.1.1

### PR DESCRIPTION
**Pull request by ndowens**
Fixes: #7088

Separated the packages. There is no license info on bashlibs, not even in the script headers

### CODE REVIEW

File: srcpkgs/cookie/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+revision=1
+_commit=9f9b6eedee297eb9148f0a5e3d9993a8b15b9f66
+create_wrksrc=yes
+build_wrksrc="${pkgname}-${version}"
+short_desc="Template-based File Generator"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://github.com/bbugyi200/cookie"
+distfiles="${homepage}/archive/v${version}.tar.gz
+ https://github.com/bbugyi200/bashlibs/archive/${_commit}.tar.gz"
```
Comment (maxice8) [inline srcpkgs/cookie/template]: shouldn't it be a separate package that cookie depends on ?

File: srcpkgs/cookie/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'cookie'
+pkgname=cookie
+version=0.1.1
+revision=1
```
Comment (maxice8) [inline srcpkgs/cookie/template]: needs noarch=yes

**Comment by maxice8** [comment]
missing dependency on `bash`

File: srcpkgs/cookie/template (hunk 1 above)
Comment (ndowens) [reply srcpkgs/cookie/template]: Can do it it, yes. Though only consists of two script files and a Makefile.

**Comment by ndowens** [comment]
Done with the changes

**Comment by ndowens** [comment]
> > Separated the packages. There is no license info on bashlibs, not even in the script headers
> 
> Then it is propietary by default, please ask upstream to license it.

Done

**Comment by maxice8** [comment]
And there is no tagged release of it.

**Comment by ndowens** [comment]
Updated cookie and bashlibs now has a version.

**Pull request merged**